configurations/manager/850GB_backup_dataset.yaml

```diff
@@ -0,0 +1,11 @@
+
+# fill the cluster with data
+prepare_write_cmd: [
+    "cassandra-stress write cl=ALL n=209715200 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=650 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..209715200",
+    "cassandra-stress write cl=ALL n=209715200 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=650 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=209715201..419430400",
+    "cassandra-stress write cl=ALL n=209715200 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=650 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=419430401..629145600",
+    "cassandra-stress write cl=ALL n=209715200 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=650 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=629145601..838860800"
+]
+
+# stress command to run
+stress_cmd_m: "cassandra-stress mixed cl=QUORUM duration=720m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate 'threads=125 fixed=20000/s' -col 'size=FIXED(1024) n=FIXED(1)'"
```

Comment:
Is ti intentional that this command doesn't use `gauss` distribution as in in the `configurations/manager/2TB_backup_dataset.yaml` config?

Reply:
i think i didn't remove this file from the PR for the purpose of future internal debug retesting, so it's not critical to leave it that way. It's not the official results we should report.